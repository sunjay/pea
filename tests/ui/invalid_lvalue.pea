fn foo() {}

fn main() {
    foo() = 3;
    abc + bcd = 5;
}
