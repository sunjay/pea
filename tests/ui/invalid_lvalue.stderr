[tests/ui/invalid_lvalue.pea:4:5-4:9] error: unsupported left-hand side of assignment expression

[tests/ui/invalid_lvalue.pea:5:5-5:13] error: unsupported left-hand side of assignment expression

error: aborting due to 2 previous errors

